labkey/security.py

```diff
@@ -242,10 +244,68 @@ def reset_password(server_context: ServerContext, email: str, container_path: st
     :return:
     """
     url = server_context.build_url(SECURITY_CONTROLLER, "adminRotatePassword.api", container_path)
-
     return server_context.make_request(url, {"email": email})
 
 
+def impersonate_user(
```

Comment:
One of the few (only?) things the python API has before any of our other language API packages. Nice.